### ISSUE TITLE
Fnf micd up modpack optimized Out now!

### ISSUE THREAD

**Issue by TikyPhase3** [luckydog7/Funkin-android#1543]
Link: www.mediafire.com/file/u91onftlrk8iuly/FnfMUModpack32Bits.apk/file

**Comment by Kappandrew** [comment]
Is there an unoptimized version, as i have a half decent device?

**Comment by TikyPhase3** [comment]
> Is there an unoptimized version, as I have a half-decent device?

No, I made this optimized and I don't want to u noptimize im taking a break.

**Comment by LarryFrosty** [comment]
why is it saying "There was a problem parsing the package"?

**Comment by TikyPhase3** [comment]
> why is it saying "There was a problem parsing the package"?

Try clearing the data and cache of the package installer then if I won't work still then force stop the package installer then try

**Comment by Ruv023** [comment]
> why is it saying "There was a problem parsing the package"?

same with me

**Comment by Ruv023** [comment]
> > why is it saying "There was a problem parsing the package"?
> 
> Try clearing the data and cache of the package installer then if I won't work still then force stop the package installer then try

Didnt work

**Comment by Kenz321** [comment]
Yeah it's saying there was a problem parsing the package for me too. Must be something up with the installation

**Comment by Caco-o-sapo** [comment]
Here too, it doesn't install in any device here. I tried in 3 different phones and none of them worked

**Comment by Kappandrew** [comment]
Yeah.... its not 64 bit. Unless your litterally on a samsung galaxy s3, this isnt going to work.

**Comment by OOOboyfriend2835** [comment]
Po

> Yeah it's saying there was a problem parsing the package for me too. Must be something up with the installation

Use file manager

**Comment by xavoeiririeiek** [comment]
Here! I made non-optimized 64bit micd up megamix! https://www.mediafire.com/file/u62l27zxft7sqfc/Funkin_MU-debug.apk/file

**Comment by Kappandrew** [comment]
> Here! I made non-optimized 64bit micd up megamix! https://www.mediafire.com/file/u62l27zxft7sqfc/Funkin_MU-debug.apk/file

Yes

**Comment by OOOboyfriend2835** [comment]
> Here! I made non-optimized 64bit micd up megamix! https://www.mediafire.com/file/u62l27zxft7sqfc/Funkin_MU-debug.apk/file

Thanks! You get a shoutout
